Formula/mu-cli.rb

```diff
@@ -10,7 +10,7 @@ class MuCli < Formula
     sha256 "8d9f97c0e15a876dda3e312bcb6a7fb13c56924ea0a8deb8c22019817bcefc21" # The Linux master sha256sum
   end
   version "1.5.10" # The master version
-  devel do
+  head do
     if OS.mac?
     url "https://github.com/stelligent/mu/releases/download/v1.5.11-develop/mu-darwin-amd64" # The MacOS develop url
     sha256 "d3804d5fbb87742ac3990160f83539769f0c9cb13ae37ada36a804f4dd25db57" # The MacOS develop sha256sum
```

Comment:
Mu is no longer being maintained. Thanks for bringing this change to our attention but I think we'd prefer to drop this block entirely.